visualizer/Readme.md

```diff
@@ -3,7 +3,8 @@
 ## Prerequisites
 
 - JDK 21 is the recommended Java runtime platform for IGV, but any release between 17 and 22 is
-  supported by the NetBeans 22 platform.
+  supported by the NetBeans 22 platform.  It's recommended that the build is performed using JDK 17
+  so that it can be used on all supported releases.
```

Comment:
Why shouldn't the build be supported on all releases? If you use compile with `--release` `javac` option, it is going to work on older JDKs regardless the JDK you compile with. Should there be a problem with that see [frgaal retrofit compiler](https://frgaal.org/#usage-with-maven). It is easy to configure from a `pom.xml` and guarantees JDK independence.

Reply:
I'll adjust the build to target release 17 and remove this language.